hack/bump-common-instancetypes.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname "$0")/common.sh
+source $(dirname "$0")/config.sh
+
+function latest_version() {
+    local org="$1"
+    local repo="$2"
+
+    # The API call sorts releases by creation timestamp, so it is enough to request only a few latest ones.
+    curl --fail -s "https://api.github.com/repos/${org}/${repo}/releases" |
+        jq '.[] | select(.prerelease==false) | .tag_name' |
+        tr -d '"' |
+        sort --version-sort |
+        tail -n1
+}
+
+val=$(latest_version "kubevirt" "common-instancetypes")
+sed -i "/^[[:blank:]]*common_instancetypes_version[[:blank:]]*=/s/=.*/=\"${val}\"/" hack/config-default.sh
```

Comment:
Won't that replace `${COMMON_INSTANCETYPES_VERSION:-"v0.3.3"}` with just the new version number?